add setup doc for BQ PSC, add beta designation (#7846)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/secure/bigquery-psc.md

---
title: "Configuring BigQuery and GCP Private Service Connect (beta)"
id: bigquery-psc
description: "Configuring GCP Private Service Connect for BigQuery (beta)"
sidebar_label: "GCP Private Service Connect for BigQuery (beta)"
---

> # Configuring BigQuery Private Service Connect <Lifecycle status="managed_plus" /> (beta)

Reviewer: ```suggestion
# Configuring BigQuery Private Service Connect <Lifecycle status="beta, managed_plus" />
```

import SetUpPages from '/snippets/_available-tiers-private-connection.md';
import CloudProviders from '/snippets/_private-connection-across-providers.md';

> <SetUpPages features={'/snippets/_available-tiers-private-connection.md'}/>

Reviewer: Not necessary to define the path twice
```suggestion
<SetUpPages />
```

> The following steps walk you through the setup of a GCP BigQuery Private Service Connect (PSC) endpoint in a <Constant name="cloud" /> multi-tenant environment.

Reviewer: ```suggestion
The following steps walk you through the setup of a GCP BigQuery [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) (PSC) endpoint in a <Constant name="cloud" /> multi-tenant environment.
```

<CloudProviders type='BigQuery' />

## Configure GCP Private Service Connect

> To enable dbt to privately connect to your BigQuery project via [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) the regional PSC endpoint will need to be enabled for your dbt account. Using the following template, submit a request to [dbt Support](/docs/dbt-support#dbt-cloud-support):

Reviewer: ```suggestion
To enable dbt to privately connect to your BigQuery project via PSC, the regional PSC endpoint needs be enabled for your dbt account. Using the following template, submit a request to [dbt Support](/docs/dbt-support#dbt-cloud-support):
```

```
Subject: New Multi-Tenant GCP PSC Request
- Type: BigQuery
- BigQuery project region: 
- dbt GCP multi-tenant environment:
```

import PrivateLinkSLA from '/snippets/_private-connection-SLA.md';

<PrivateLinkSLA />

> ## Create Connection in dbt

Reviewer: ```suggestion
## Create the connection in dbt
```

> Once <Constant name="cloud" /> support completes the configuration, you can start creating new connections using PrivateLink. 

Reviewer: ```suggestion
Once the dbt Support team completes the configuration, you can start creating new connections using PSC: 
```
